### ISSUE TITLE
"Transitivity works on simple graphs only" error has incorrect information

### ISSUE THREAD

**Issue by szhorvat** [igraph/rigraph#491]
When trying to compute local transitivity on a multigraph, we see this message:

```
> g<-graph_from_literal(1-2,1-2,2-3, simplify=F)
> transitivity(g, type = 'local')
Transitivity works on simple graphs only. The result might be incorrect. igraph 1.3.0 and later will treat this as an error.
```

Notice "igraph 1.3.0 and later will treat this as an error."

R/igraph 1.3 will be based on C/igraph 0.9, correct? In the most recent 0.9 version, all transitivities except the weighted one is supported for multigraphs. As I recall, this was not the case for 0.9.0, but we fixed it somewhere along the way. Now edge multiplicities are simply ignored.

**Comment by ntamas** [comment]
I added that part to the error message at a time when it was not sure whether igraph 1.3.0 will be based in C-igraph 0.9.x or 0.8.x and I wanted to be on the safe side. Nevertheless, I leave this issue open because we need to deal with the text of the warning message either way; if 1.3.0 will still be based on 0.8.x, then we need to change this into an error; if 1.3.0 will be based on 0.9.x, we need to remove the message altogether.

**Comment by szhorvat** [comment]
Yes, I opened this as a reminder. These things are easy to forget.

**Comment by szhorvat** [comment]
This is already fixed for 1.3, which will be based on 0.9. Closing.
